test/scripts/atomic-swap-teal-test.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+date '+atomic-swap-teal-test start %Y%m%d_%H%M%S'
+
+set -e
+set -x
+
+TEMPDIR=$(mktemp -d)
```

Comment:
see Tsachi's recent change to the similar scripts about setting GOPATH near the top of these scripts, and yes, we'll clean it up properly post release

Reply:
done, thanks